css/home.css

```diff
@@ -69,21 +69,30 @@ body {
 }
 
 .homepage-hero__article {
-  margin-right: 5%;
-  margin-left: 5%;
+  margin-right: 57px;
+  margin-left: 57px;
+  position: absolute;
+  bottom: 250px;
+  top: 250px;
+}
+.homepage-hero__lookdown{
```

Comment:
Please look how to center absolute positioned elements, this icon is not centered in dektop view. Also search how to center the logo vertically and horizontally.

Reply:
Please add a space between the selector and the bracket.